{{cookiecutter.project_slug}}/setup.cfg

```diff
@@ -1,12 +1,40 @@
 [metadata]
-name = {{cookiecutter.project_slug}}
+name = {{ cookiecutter.project_slug }}
+description = {{ cookiecutter.package_description }}
+long_description = file: README.md
+long_description_content_type = text/markdown; charset=UTF-8
+author = Thermondo GmbH
+author-email = opensource@thermondo.de
```

Comment:
isn't this only for open source packages? we don't opensource everything.

Reply:
Yes, because that would be the default case. By default what we build is open source unless it contains a USP or IP.